src/main.rs

```diff
@@ -41,6 +41,18 @@ enum Command {
         package: PackageId,
     },
 
+    /// Packages this api
+    #[clap(alias = "pack")]
+    Package {
+        /// Destination directory for the release
+        #[clap(long)]
+        #[arg(default_value = ".")]
+        directory: String,
```

Comment:
Can we name it `--output-directory`? Directory could refer to the package directory as well / could be misleading

Reply:
Makes sense, will update